tests/integration/baseImage_test.go

```diff
@@ -25,6 +25,10 @@ func TestBaseImage(t *testing.T) {
 	if len(eveBaseRef) == 0 {
 		zArch = "amd64"
 	}
+	HV := os.Getenv("HV")
```

Comment:
I'm fine with merging it for now, but we have to very quickly decide how to handle these types of config generically @sadov 

I filed https://github.com/lf-edge/eden/issues/10 to figure that one out -- please take a look

Ultimately, whether baseOS version contains a hypervisor label or not (and what kind) is completely orthogonal to what the test does. We have to have a way of making sure we can test any combination and I think that that can only be driven by external configuration.